### PR TITLE
Added the file "el2731"

### PR DESCRIPTION
**Pull request by el2731**
This adds the file el2731 because it introduces me.

### CODE REVIEW

File: Participants/el2731 [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Name: Evan Lin
+Class year: 2026 (Freshman)
+Odd fact: I took four years of Latin and did not have to study hard for any exams
```
Comment (ianfridman) [inline Participants/el2731:3]: Impressive! Lux et veritas for you!

**Review by ianfridman — CHANGES_REQUESTED** [review]
Add the word "great" before class year.

File: Participants/el2731 (hunk 1 above)
Comment (el2731) [reply Participants/el2731:3]: Thank you! Same to you!

**Comment by el2731** [comment]
I made the changes. Please review again! Thank you!

**Review by ianfridman — APPROVED** [review]
Looks good.

**Pull request merged**